keyboards/3.0/xct-Tibt-t-k0-qwerty.xml

```diff
@@ -0,0 +1,494 @@
+      <key id="N"     output="\u{0F4E}" />
+      <key id="t"     output="\u{0F4F}" />
+      <key id="T"     output="\u{0F50}" />
+      <key id="d"     output="\u{0F51}" />
+      <key id="n"     output="\u{0F53}" />
+      <key id="p"     output="\u{0F54}" />
+      <key id="P"     output="\u{0F55}" />
+      <key id="b"     output="\u{0F56}" />
+      <key id="m"     output="\u{0F58}" />
+      <key id="w"     output="\u{0F5D}" />
+      <key id="v"     output="\u{0F5D}" />
+      <key id="Z"     output="\u{0F5E}" />
+      <key id="z"     output="\u{0F5F}" />
+      <key id="apos"  output="\u{0F60}" />
+      <key id="y"     output="\u{0F61}" />
+      <key id="Y"     output="\m{fullYa}"/>
+      <key id="r"     output="\u{0F62}" />
+      <key id="R"     output="\u{0F6A}" />
+      <key id="l"     output="\u{0F63}" />
+      <key id="alt_s" output="\u{0F64}" />
+      <key id="S"     output="\u{0F65}" />
+      <key id="alt_x" output="\u{0F65}" />
+      <key id="s"     output="\u{0F66}" />
+      <key id="h"     output="\u{0F67}" />
+
+    <!-- specials -->
+      <key id="f"     output="\m{STACK}" />
+      <key id="cycle" output="\m{CYCLE}" />
+
+    <!-- digits -->
+      <key id="1"     output="\u{0F21}" />
+      <key id="2"     output="\u{0F22}" />
+      <key id="3"     output="\u{0F23}" />
+      <key id="4"     output="\u{0F24}" />
+      <key id="5"     output="\u{0F25}" />
+      <key id="6"     output="\u{0F26}" />
+      <key id="7"     output="\u{0F27}" />
+      <key id="8"     output="\u{0F28}" />
+      <key id="9"     output="\u{0F29}" />
+      <key id="0"     output="\u{0F20}" />
+
+    <!-- digits minus half -->
+      <key id="alt_1" output="\u{0F2A}" />  
+      <key id="alt_2" output="\u{0F2B}" />  
+      <key id="alt_3" output="\u{0F2C}" />  
+      <key id="alt_4" output="\u{0F2D}" /> 
+      <key id="alt_5" output="\u{0F2E}" /> 
+      <key id="alt_6" output="\u{0F2F}" /> 
+      <key id="alt_7" output="\u{0F30}" /> 
+      <key id="alt_8" output="\u{0F31}" /> 
+      <key id="alt_9" output="\u{0F32}" /> 
+      <key id="alt_0" output="\u{0F33}" /> 
+  </keys>
+
+  <layers formId="us">
+    <layer modifiers="none">
+      <row keys="tilde 1 2 3 4 5 6 7 8 9 0 hyphen equal"/>
+      <row keys="q w e r t y u i o p open-square close-square pipe"/>
+      <row keys="a s d f g h j k l gtertsheg apos"/>
+      <row keys="z x c v b n m comma period cycle"/>
+      <row keys="tsheg"/>
+    </layer>
+    <layer modifiers="shift">
+      <row keys="tilde bang at hash dollar percent caret amp asterisk open-paren close-paren underscore plus"/>
+      <row keys="Q W E R T Y U I O P open-curly close-curly pipe"/>
+      <row keys="A S D F G H J K L colon double-quote"/>
+      <row keys="Z X C V B N M open-angle close-angle question"/>
+      <row keys="space"/>
+    </layer>
+    <layer modifiers="caps">
+      <row keys="yigmgo dzudrtags cap_2 cap_3 cap_4 cap_5 cap_6 cap_7 cap_8 9 0 chadrtags lhagrtags"/>
```

Comment:
Curious to me that we have cap_2 .. cap_8 but 9 and 0 are unmodified; this may feel a little weird to a user, if there are just two keys on this row that drop back to 'base layer'.

Reply:
Added cap_9 and cap_0